examples/pubsub/tutorial_pubsub_mqtt_publish.c

```diff
@@ -459,10 +461,12 @@ int main(int argc, char **argv) {
      * the pubsub connection tutorial */
     UA_ServerConfig_setDefault(config);
     UA_ServerConfig_addPubSubTransportLayer(config, UA_PubSubTransportLayerMQTT());
-
+    
+    UA_NodeId fieldNodeId;
+    memset(&fieldNodeId, 0, sizeof(UA_NodeId));
```

Comment:
Please use UA_NodeId_init in the example.

Reply:
Done